### PR TITLE
Add annotations for explicit declaration

### PR DESCRIPTION
**Pull request by joongwon**
Please review

### CODE REVIEW

File: library/src/main/java/com/evernote/android/state/StateSaver.java [hunk 1]
```diff
@@ -32,16 +32,19 @@
 
     private static final Map<Class<?>, Injector> INJECTORS = new LinkedHashMap<>();
 
-    private static Injector getInjector(Class<?> cls)
+    @Nullable
+    private static Injector getInjector(@NonNull Class<?> cls)
             throws IllegalAccessException, InstantiationException {
-        Injector injector = INJECTORS.get(cls);
-        if (injector != null || INJECTORS.containsKey(cls)) {
```
Comment (vRallev) [inline library/src/main/java/com/evernote/android/state/StateSaver.java]: This was a small optimization. `INJECTORS` could contain a null value, so the `containsKey` check is necessary, see #31

File: library/src/main/java/com/evernote/android/state/StateSaver.java [hunk 2]
```diff
@@ -53,7 +56,8 @@ private static Injector getInjector(Class<?> cls)
     }
 
     @SuppressWarnings("unchecked")
-    private static <T extends Injector> T safeGet(Object target, Injector nop) {
+    @NonNull
+    private static <T extends Injector> T safeGet(@NonNull Object target, @NonNull Injector nop) {
```
Comment (vRallev) [inline library/src/main/java/com/evernote/android/state/StateSaver.java]: That's an internal method, but I would merge these two lines nonetheless

File: library/src/main/java/com/evernote/android/state/StateSaver.java (hunk 1 above)
Comment (joongwon) [reply library/src/main/java/com/evernote/android/state/StateSaver.java]: I know, but NPE check is duplicated in `safeGet` method.
So I don't think it's necessary.

Comment (vRallev) [reply library/src/main/java/com/evernote/android/state/StateSaver.java]: It is, because then it saves the expensive reflection call.

**Comment by vRallev** [comment]
Closing because of inactivity.

**Pull request closed without merging**